<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fpd gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1050</width>
        <height>625</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="home_tab">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QGroupBox" name="workflow_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Workflow help</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="log_text">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string>Start by loading files</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="mid_hSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QGroupBox" name="functions_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="title">
           <string>Functions</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QPushButton" name="ransac_tools_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Ransac Tools</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="centre_of_mass_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string> Center of mass</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButton_42">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Dummy</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="dbrowser_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Data Browser</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="rm_aperture_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Synthetic aperture</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="find_circ_c_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Find circular center</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="dpc_explorer_button">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>DPC Explorer</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_41">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="text">
              <string>Dummy</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="file_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="title">
           <string>File opening</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="mib_button">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>.mib</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dm3_button">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>.dm3</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="hdf5_button">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>.hdf5</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="mib_line">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>.mib filename</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dm3_line">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>.dm3 filename</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hdf5_line">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>.hdf5 filename</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="load_button">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Load</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clear_button">
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                 <underline>false</underline>
                </font>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <widget class="QGroupBox" name="resource_groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
           </font>
          </property>
          <property name="title">
           <string>Resource monitor</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="CpuFreqGraph" name="resMonitor" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="right_hSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="left_hSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="dark_mode_button">
      <property name="text">
       <string>Dark Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="action_mib"/>
     <addaction name="action_dm3"/>
     <addaction name="action_hdf5"/>
    </widget>
    <addaction name="menuOpen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_dm3">
   <property name="text">
    <string>.dm3</string>
   </property>
  </action>
  <action name="action_mib">
   <property name="text">
    <string>.mib</string>
   </property>
  </action>
  <action name="action_hdf5">
   <property name="text">
    <string>.hdf5</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CpuFreqGraph</class>
   <extends>QWidget</extends>
   <header>fpd_explorer.resource_monitor</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mib_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>function_mib()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbrowser_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_dbrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>727</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>find_circ_c_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>find_circular_centre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>778</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>load_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dm3_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>function_dm3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear_files()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dark_mode_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change_color_mode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>684</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>720</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rm_aperture_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_aperture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centre_of_mass_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>centre_of_mass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dpc_explorer_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_dpc_explorer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>993</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>1094</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_files()</slot>
  <slot>function_mib()</slot>
  <slot>function_dm3()</slot>
  <slot>start_dbrowser()</slot>
  <slot>find_circular_centre()</slot>
  <slot>clear_files()</slot>
  <slot>change_color_mode()</slot>
  <slot>remove_aperture()</slot>
  <slot>centre_of_mass()</slot>
  <slot>start_dpc_explorer()</slot>
 </slots>
</ui>
